src/server/middleware.js

```diff
@@ -34,8 +34,8 @@ export default function (configDir) {
     res.send(getIndexHtml(publicPath));
   });
 
-  const headHtml = getHeadHtml(configDir);
   router.get('/iframe.html', function (req, res) {
+    const headHtml = getHeadHtml(configDir);
```

Comment:
This cause performance issues. 
But since this is a dev only tool, this is totally fine.